modules/ossm-smcp-prod.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deploy-production.adoc
+
+
+[id="ossm-smcp-prod_{context}"]
+= Configuring your `ServiceMeshControlPlane` resource for production
+
+If you have installed a basic `ServiceMeshControlPlane` resource to test {ProductShortName}, you must configure it to production specifications before your deploy your production service mesh.
+
+You cannot change the `metadata.name` field of an existing `ServiceMeshControlPlane` resource. 
+ 
+For production deployments you _must_ customize the default template. Follow this procedure to find and save your staging `ServiceMeshControlPlane` resource by using the CLI.
+
+Follow this procedure to create or edit the `ServiceMeshControlPlane` with the command line.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command. Then, enter your username and password when prompted.
++
+[source,terminal]
+----
+$ oc login https://{HOSTNAME}:6443
+----
++
+. Change to the project where you installed the control plane, for example istio-system.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
++
+. Retrieve the current `ServiceMeshControlPlane` resource and save it locally to edit it.
++
+[source,terminal]
+----
+oc get smcp <smcp_name> -o yaml > <smcp_name>.v2.yaml
+----
++
+. Edit the `ServiceMeshControlPlane` file. Change the `metadata.name` field to `production`.
++
+. Configure Jaeger for production. To use the `production` deployment strategy, set  `spec.addons.jaeger.install.storage.type` to 'Elasticsearch' and specify additional configuration options under `install`. Or, you can create and configure your Jaeger instance and set `spec.addons.jaeger.name` to the name of the Jaeger instance, for example `jaeger-production`.
++
+.Control Plane default Jaeger parameters (Elasticsearch)
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: production
```

Comment:
We might consider changing the name here:
1. It can't be changed
2. It implies the mesh is for production
3. This spec could be used anywhere, for things like dev, qa, etc.

I'd stick with things like: mesh, default, example, ...

Reply:
@rcernich I like this idea, and it was one of the things I was wondering. 

Should I also be considering changing the name in the installation guide? https://docs.openshift.com/container-platform/4.7/service_mesh/v2x/installing-ossm.html#ossm-control-plane-deploy-cli_installing-ossm

So instead of naming it basic there, we could name it default, and then we can get rid of the steps in this item to rename metadata.name to production. We'd just start these steps with the Jaeger stuff. It seems like a lot of work to change the name, and changing the name has a really low value it seems to me. Configuring Jaeger and making sure the security certs are set up properly for production seems like much more of a high value task to me.

Reply:
I think that's a good approach.  The user is really just applying production ready settings to their SMCP vs. something that's maybe more geared toward development use (e.g. higher trace sampling is probably more useful in dev than in prod; in dev, you're more likely to need tracing to help figure out what's going on, but there's a performance penalty, so you probably don't want to run that in prod).